packages/patternlab/styleguide/source/_patterns/02-molecules/image-promo.twig

```diff
@@ -87,7 +77,9 @@
 
     {% block imagePromoLink %}
       {% if imagePromo.link.text and not imagePromo.location.text %}
-        <div class="ma__image-promo__link">
+        {# Hide more link from AT users.
+           This link is redundant since the entire container is clickable to go to the same desitnation. #}
+        <div class="ma__image-promo__link" aria-hidden="true">

```

Comment:
I think we should not render this link for all users. This goes to the exact same URL as the whole teaser, hence the link the redundant for all users.

Reply:
My local change of commenting out the entire block was not pushed somehow the other day.  This link wouldn't get rendered at all.